css/index.css

```diff
@@ -211,9 +215,26 @@ footer p {
   height: 174.75px;
   border-radius: 20px;
 }
+@media - (max-width:500px) {
+  .container .three-imgs .img:first-child {
+    display: block;
+    width: 85%;
+    height: 300px;
+  }
+}
+@media (max-width:500px) {
+  .container .three-imgs .middle {
```

Comment:
need to add img to the start of the .middle to give more specificity so it should read img.middle